### PR TITLE
 "test_multiple_pvc_deletion_measurement_performance" test case impro…

### PR DESCRIPTION
**Pull request by pintojoy**
"test_multiple_pvc_deletion_measurement_performance" test case improvement to create multiple PVCs in seperate namespace instead of openshift-storage namespace.

This will create a new namespace, create 120PVCs in the new name space and once test is completed newly created namespace would be deleted.

### CODE REVIEW

File: tests/e2e/performance/test_pvc_creation_deletion_performance.py [hunk 1]
```diff
@@ -105,6 +106,15 @@ def base_setup(self, interface_type, storageclass_factory, pod_factory):
             self.sc_obj = storageclass_factory(self.interface)
         self.pod_factory = pod_factory
 
+    @pytest.fixture()
+    def namespace(self, project_factory):
+        """
+        Create a new project
+        """
+        self.start_time = default_timer()
```
Comment (Avilir) [inline tests/e2e/performance/test_pvc_creation_deletion_performance.py:114]: what is the purpose of this ?

Comment (keesturam) [inline tests/e2e/performance/test_pvc_creation_deletion_performance.py:114]: Do you use this somewhere else?

File: tests/e2e/performance/test_pvc_creation_deletion_performance.py [hunk 2]
```diff
@@ -105,6 +106,15 @@ def base_setup(self, interface_type, storageclass_factory, pod_factory):
...
+        proj_obj = project_factory()
+        self.namespace = proj_obj.namespace
+
```
[the remark this answers is not in the document]
Comment (pintojoy) [reply tests/e2e/performance/test_pvc_creation_deletion_performance.py:117]: Namespace would be deleted at the end of the test:
15:43:55 - MainThread - ocs_ci.ocs.ocp - INFO - Switching to project openshift-storage
15:43:55 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc project openshift-storage
15:43:55 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0635fb85401745dbb967746e3 delete Project namespace-test-0635fb85401745dbb967746e3
15:43:56 - MainThread - ocs_ci.utility.utils - WARNING - Command stderr: warning: deleting cluster-scoped resources, not scoped to the provided namespace

15:43:56 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0635fb85401745dbb967746e3 get Project namespace-test-0635fb85401745dbb967746e3 -n namespace-test-0635fb85401745dbb967746e3 -o yaml
15:44:00 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0635fb85401745dbb967746e3 get Project namespace-test-0635fb85401745dbb967746e3 -n namespace-test-0635fb85401745dbb967746e3 -o yaml
15:44:03 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0635fb85401745dbb967746e3 get Project namespace-test-0635fb85401745dbb967746e3 -n namespace-test-0635fb85401745dbb967746e3 -o yaml
15:44:06 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-0635fb85401745dbb967746e3 get Project namespace-test-0635fb85401745dbb967746e3 -n namespace-test-0635fb85401745dbb967746e3 -o yaml
15:44:07 - MainThread - ocs_ci.utility.utils - WARNING - Command stderr: Error from server (NotFound): namespaces "namespace-test-0635fb85401745dbb967746e3" not found

File: tests/e2e/performance/test_pvc_creation_deletion_performance.py (hunk 1 above)
Comment (pintojoy) [reply tests/e2e/performance/test_pvc_creation_deletion_performance.py:114]: Sure Ram. I will remove it

Comment (pintojoy) [reply tests/e2e/performance/test_pvc_creation_deletion_performance.py]: I have removed default_timer reference

Comment (pintojoy) [reply tests/e2e/performance/test_pvc_creation_deletion_performance.py]: I have removed default_timer reference

**Review by keesturam — APPROVED** [review]
LGTM

**Review by Avilir — APPROVED** [review]
LGTM

**Pull request merged**